test/utils/__snapshots__/setupHooks.test.js.snap

```diff
@@ -1,39 +1,9 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`setupHooks handles multi compiler 1`] = `
-Array [
-  Array [
-    "Child \\"comp1\\": Failed to compile.",
-  ],
-  Array [
-    "Child \\"comp2\\": Compiled with warnings.",
-  ],
-]
-`;
+exports[`setupHooks handles multi compiler 1`] = `Array []`;
```

Comment:
so... we need to fix these tests(an empty array) later, right?

Reply:
I want to fix it here (improve tests), but don't have ideas how better to test it, maybe mocking console?

Reply:
sorry for the late reply, yeah, I think we can mock the console.

Reply:
I think I will rewrite logging tests on execa, and will look at stdout and stderr